cfme/tests/automate/test_customization_paginator.py

```diff
@@ -66,6 +67,10 @@ def test_paginator(some_dialogs, soft_assert):
         for text in get_relevant_rows(common.dialogs_table):
             dialogs_found.add(text)
         current_rec_offset = paginator.rec_offset()
+
+        new_total = paginator.rec_total()
+        assert current_total == new_total, \
+            "Total value has changed, expected {0}, but was {1}".format(current_total, new_total)
```

Comment:
Yeh I'd agree with @mfalesni here, the dynamic nature of this means it really could change, unless we have a provider that's a dummy.....